dallinger/experiment.py

```diff
@@ -68,8 +71,11 @@ def __init__(self, session=None):
         self.experiment_repeats = 0
 
         #: Recruiter, the Dallinger class that recruits participants.
-        #: Default is PsiTurkRecruiter.
-        self.recruiter = PsiTurkRecruiter
+        #: Default is HotAirRecruiter in debug mode and PsiTurkRecruiter in other modes.
+        if config.get('mode') == 'debug':
+            self.recruiter = HotAirRecruiter
+        else:
+            self.recruiter = PsiTurkRecruiter
```

Comment:
My branch moves this into a property on the class, because we can't really rely on the config having been loaded at `__init__` time. Additionally, it may be useful for existing instances to respond to config changes. In the long run we need a better way to configure recruiters as we get more of them.